code/src/java/pcgen/rules/context/TrackingManufacturer.java

```diff
@@ -286,6 +286,6 @@ public String unconvert(T arg0)
 	@Override
 	public boolean isDirect()
 	{
-		return false;
+		return rm.isDirect();
```

Comment:
This sticks out as different. I do not see 'rm' defined in the changes. Is this intentional?

Reply:
rm is set in the initializer method as a ReferenceManufacturer that is passed in.  This looks fine.